src/media/css/header.styl

```diff
@@ -229,6 +229,10 @@ body > .banners mkt-banner {
     }
 }
 
+[data-page-type~="leaf"] .subheader {
```

Comment:
Can you update these to include `[data-page-type~="leaf"] .secondary-header` as well? That should fix the Report an Issue and Report Abuse pages.